### DISCUSSION TITLE
usbguard package

### DISCUSSION THREAD

**Discussion by Peteri42** [USBGuard/usbguard#605] [Q&A] [answered]
Can i ask why when i install the usbguard package from some manager (apt or zypper) it does install the .so but not the .pc so CMAKE can't find anything when i search for it but when i install it from this repo it does install the .pc file.

**Comment by muelli** [comment]
usually, development files are not part of the ordinary release but rather part of the "development" part of the package.
You can use, e.g. packages.ubuntu.com to look for your desired file. You'll find it here: https://packages.ubuntu.com/mantic/amd64/libusbguard-dev/filelist

**Reply by Peteri42** [reply]
I haven't been using linux for long is it possible to add this package to my package manager and then download it through it or do i have to install it manually?

**Reply by Peteri42** [reply]
I'm sorry to bother you further but i'd appriciate the help. I installed the usbguard CLI and the dev package. But when i try to run the CLI that i install from here it says "ERROR: IPC connect: service=usbguard: Connection refused" but when i use the one from apt it works just fine. Any idea what i could be doing wrong?

**Reply by radosroka** [reply]
Did you generate rules for the daemon and started it?

**Reply by Peteri42** [reply]
![image](https://github.com/USBGuard/usbguard/assets/135226956/b0dee3c4-2a72-45d4-8111-168fc183623a)
I generated rules and started and enabled the daemon.

**Reply by Peteri42** [reply]
I started both usbguard-dbus and usbguard-service

**Reply by Peteri42** [reply]
I managed to fix my problem the CLI now communicates with the service but i can't seem to launch the usbguard-dbus.service even if i tried to install it on a fresh ubuntu system. Always throws the same error.
![image](https://github.com/USBGuard/usbguard/assets/135226956/380afa2b-c1d2-4485-9531-06ebbb817110)
Do i need to change something in the config files ? or maybe i'm missing some package?

**Reply by Cropi** [reply]
Aren't there any messages in the journal indicating the root cause? When you face a `Start request repeated too quickly. Failed with tesult 'exit-code'` error, you will have to check the error message before. There should be some.

**Reply by Peteri42** [reply]
![image](https://github.com/USBGuard/usbguard/assets/135226956/c37ff3fc-0341-4bc5-8104-cf167d137c6e)
This is all it says.

**Reply by Peteri42** [reply]
I tried running it on a fresh ubuntu VM and i run into the same problem. I install all packages that are in readme, run ./configure with systemd, install it, generate rules and the usbguard service itself works even the CLI but i can't seem to get the dbus running. Is there something else i'm missing?

**Reply by radosroka** [reply, marked as answer]
Why not to use official usbguard package?
